cmd/eks.go

```diff
@@ -320,3 +342,23 @@ func syncClusterComponents(cliContext *cli.Context) error {
 	waitTimeout := cliContext.String(waitTimeoutFlag.Name)
 	return eks.SyncClusterComponents(eksClusterArn, shouldWait, waitTimeout)
 }
+
+// Command action for `kubergrunt eks cleanup-security-group`
+func cleanupSecurityGroup(cliContext *cli.Context) error {
+	eksClusterArn, err := entrypoint.StringFlagRequiredE(cliContext, "eks-cluster-arn")
+	if err != nil {
+		return errors.WithStackTrace(err)
+	}
+
+	securityGroupID, err := entrypoint.StringFlagRequiredE(cliContext, "security-group-id")
+	if err != nil {
+		return errors.WithStackTrace(err)
+	}
```

Comment:
NIT: If you agree with the ergonomics question above, this should be adjusted to be optional.

Reply:
(filed as a nice to have)